tests/typescript.test.js

```diff
@@ -10,8 +10,9 @@ describe.concurrent('Test typescript blueprint on Ember', function () {
         expect,
         version,
         packages,
-        '--typescript --skip-npm --pnpm',
+        ['--typescript', '--skip-npm', '--pnpm'],
```

Comment:
We need this array format for the options to be parsed correctly. The one-string shape was passed to ember-cli as only one invalid option, so the generated app was not in TypeScript.